xl/version.py

```diff
@@ -41,7 +41,7 @@ def get_current_revision(directory):
     try:
         return subprocess.check_output([
             'git', 'rev-parse', '--short=7', 'HEAD'
-        ]).strip()
+        ], stderr=-1).strip()
```

Comment:
I'm curious, the documentation doesn't state what happens when you set stderr to an invalid file descriptor...